src/views/Dashboard/Customer/Home/GeneticData/RequestAnalysis/ServiceAnalysisCard.vue

```diff
@@ -50,10 +65,33 @@ export default {
 
   computed: {
     ...mapState({
+      api: (state) => state.substrate.api,
       web3: (state) => state.metamask.web3
     })
   },
 
+  async mounted() {
+
+    if(this.$route.params.id) {
+      this.isCreated = true
+      this.orderId = this.$route.params.id
+      await this.getServiceDetail()
+    }
+
+    this.serviceName = this.service.serviceName
+    this.description = this.service.description
+    this.duration = this.service.duration
+    this.durationType = this.service.durationType
+    this.price = `${this.formatBalance(this.service.priceDetail[0].totalPrice)} ${this.service.priceDetail[0].currency}`
+
+    if (!this.service.analystsInfo.info.profielImage) {
+      this.profileImage = "https://ipfs.debio.network/ipfs/QmQ9Zvn4pMNMr3nVTGzYE9zGQfyFrhyAHWZNjm76pJmvRn"
```

Comment:
The DeBio logo shouldn't be hardcoded like this. Wouldn't it be better to turn it into an Icon?

Please ask @bukandicki for help. @bukandicki, please help @ciniiia with this.